python/ray/async_compat.py

```diff
@@ -7,9 +7,22 @@
 import time
 import inspect
 
+try:
+    import uvloop
+except ImportError:
+    uvloop = None
+
 import ray
 
 
+def get_new_event_loop():
+    """Construct a new event loop. Ray will use uvloop if it exists"""
+    if uvloop:
```

Comment:
Should we also be making uvloop a dependency of serve?

Reply:
uvloop is already a dependency of serve via `uvicorn`. the `uv` in the `uvicorn` means `uvloop`, so we are covered.

Reply:
It looks like `uvloop` is not a dependency of `uvicorn` but only of `uvicorn[standard` as per the documentation of `uvicorn`: 

> $ pip install uvicorn
> 
> This will install uvicorn with minimal (pure Python) dependencies.
> 
> $ pip install uvicorn[standard]
> 
> This will install uvicorn with "Cython-based" dependencies (where possible) and other "optional extras".
> 
> In this context, "Cython-based" means the following:
> 
>     the event loop uvloop will be installed and used if possible.

https://www.uvicorn.org/#quickstart

Reply:
Ah you already had one!